### PR TITLE
css-colors - Kyle Kettler

### PR DESCRIPTION
**Pull request by kyle-kettler**

### CODE REVIEW

**Comment by kyle-kettler** [comment]
Screenshot 2023-05-16 at 3 49 04 PM

File: css-colors/css-colors-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# css-colors-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Name three different types of values you can use to specify colors in CSS.
+
```
Comment (shawnkost) [inline css-colors/css-colors-quiz-notes.md:8]: Don't forget the quiz questions!

File: css-colors/styles.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: rgb(244, 249, 251);
```
Comment (shawnkost) [inline css-colors/styles.css:8]: This should be applied to the `section` instead

**Review by shawnkost — CHANGES_REQUESTED** [review]
A couple changes requested here Kyle!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**